### PR TITLE
White-label support + enhanced license validation panel

### PR DESCRIPTION
**Pull request by glueckpress**
- removed all branding (incl. docs links) from options when WL is enabled
- enhanced license validation panel

### CODE REVIEW

**Comment by glueckpress** [comment]
@GeekPress The support does not seem to send anything when the license is not active, that’s why I removed the support tab there. I tested multiple times, only saw a perpetuating sending animation, but my message did not get transmitted.

**Comment by glueckpress** [comment]
@GeekPress Hold on, it may have been my bad. I had actually removed the license data from my install. That would hardly be the case for a customer, admittedly. 

Can we confirm the support form works even when the connection to our servers does not work for license validation?

File: inc/admin/options.php [hunk 1]
```diff
@@ -885,8 +883,8 @@ function rocket_settings_callback( $inputs ) {
 		$inputs['cloudflare_domain'] = '';
 	}
 
-	$inputs['cloudflare_devmode'] = ( is_numeric( $inputs['cloudflare_devmode'] ) ) ? (int) $inputs['cloudflare_devmode'] : 0;
-	$inputs['cloudflare_auto_settings'] = ( is_numeric( $inputs['cloudflare_auto_settings'] ) ) ? (int) $inputs['cloudflare_auto_settings'] : 0;
+	$inputs['cloudflare_devmode'] = ( isset( $inputs['cloudflare_devmode'] ) && is_numeric( $inputs['cloudflare_devmode'] ) ) ? (int) $inputs['cloudflare_devmode'] : 0;
+	$inputs['cloudflare_auto_settings'] = ( isset( $inputs['cloudflare_auto_settings'] ) && is_numeric( $inputs['cloudflare_auto_settings'] ) ) ? (int) $inputs['cloudflare_auto_settings'] : 0;
```
Comment (Tabrisrp) [inline inc/admin/options.php]: this should alway be set, did you encounter a case that require this additional check?

File: inc/admin/ui/modules/cdn.php [hunk 2]
```diff
@@ -137,7 +141,7 @@
 			'type'         => 'textarea',
 			'label_for'    => 'cdn_reject_files',
 			'label_screen' => __( 'Exclude files:', 'rocket' ),
-			'placeholder'  => "e.g. /wp-content/themes/some-theme/style.css\ne.g. /wp-content/plugins/some-plugin/(.*).css"
+			'placeholder'  => "/wp-content/themes/some-theme/style.css\n/wp-content/plugins/some-plugin/(.*).css"
```
Comment (Tabrisrp) [inline inc/admin/ui/modules/cdn.php]: \n doesn't work in the placeholder

File: inc/admin/ui/modules/optimization.php [hunk 3]
```diff
@@ -202,7 +216,7 @@
 			'type'         => 'textarea',
 			'label_for'    => 'exclude_css',
 			'label_screen' => __( 'CSS files to be excluded from minification and concatenation:', 'rocket' ),
-			'placeholder'  => "e.g. /wp-content/themes/some-theme/style.css\ne.g. /wp-content/plugins/some-plugin/(.*).css",
+			'placeholder'  => "/wp-content/themes/some-theme/style.css\n/wp-content/plugins/some-plugin/(.*).css",
```
Comment (Tabrisrp) [inline inc/admin/ui/modules/optimization.php]: Same here \n doesn't work

File: inc/admin/ui/modules/optimization.php [hunk 4]
```diff
@@ -233,7 +247,7 @@
 			'type'         => 'textarea',
 			'label_for'    => 'exclude_js',
 			'label_screen' => __( 'JS files to be excludeded from minification and concatenation:', 'rocket' ),
-			'placeholder'  => "e.g. /wp-content/themes/some-theme/(.*).js\ne.g. /wp-content/plugins/some-plugin/script.js",
+			'placeholder'  => "/wp-content/themes/some-theme/(.*).js\n/wp-content/plugins/some-plugin/script.js",
```
Comment (Tabrisrp) [inline inc/admin/ui/modules/optimization.php]: + here

File: inc/admin/options.php (hunk 1 above)
Comment (glueckpress) [inline inc/admin/options.php]: Yes, when my CF credentials when I saved CF settings with a wrong API key and wrong email, I got a warning without those `isset()`s.

File: inc/admin/ui/modules/cdn.php (hunk 2 above)
Comment (glueckpress) [inline inc/admin/ui/modules/cdn.php]: [MDN reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) wasn’t very clear about it, but [spec is](https://www.w3.org/TR/html5/forms.html#the-placeholder-attribute), so changing it.

Comment (glueckpress) [inline inc/admin/ui/modules/cdn.php]: Done in https://github.com/wp-media/wp-rocket/pull/418/commits/6696b048b0799dbd6882b820ee493ae18814fa38

File: inc/admin/ui/modules/optimization.php (hunk 3 above)
Comment (glueckpress) [inline inc/admin/ui/modules/optimization.php]: Done in https://github.com/wp-media/wp-rocket/pull/418/commits/6696b048b0799dbd6882b820ee493ae18814fa38

File: inc/admin/ui/modules/optimization.php (hunk 4 above)
Comment (glueckpress) [inline inc/admin/ui/modules/optimization.php]: Done in https://github.com/wp-media/wp-rocket/pull/418/commits/6696b048b0799dbd6882b820ee493ae18814fa38

**Pull request merged**